<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="USER-PC" name="Multiple Failures Test" tests="3" failures="1" timestamp="19 Oct 2015 11:59:37 GMT" time="0.144" errors="0">
  <testcase name="multipleFailures" time="0.039" classname="framework.datadriven.stockSuite.MultipleFailures">
    <failure type="java.lang.AssertionError" message="expected [B3] but found [A3]">
      <![CDATA[java.lang.AssertionError: expected [B3] but found [A3]
	at framework.datadriven.stockSuite.MultipleFailures.multipleFailures(MultipleFailures.java:50)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- multipleFailures -->
  <testcase name="multipleFailures" time="0.038" classname="framework.datadriven.stockSuite.MultipleFailures">
    <skipped/>
  </testcase> <!-- multipleFailures -->
  <testcase name="multipleFailures" time="0.036" classname="framework.datadriven.stockSuite.MultipleFailures">
    <skipped/>
  </testcase> <!-- multipleFailures -->
</testsuite> <!-- Multiple Failures Test -->
